src/seismometer/_api.py

```diff
@@ -148,14 +157,12 @@ def target_feature_summary(exclude_cols: list[str] = None, inline=False):
 
 
 @export
-def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1.25) -> None:
+def fairness_audit(metric_list: Optional[list[str]] = None, fairness_threshold=1.25) -> HTML:
```

Comment:
for consistency, would it make sense to have a typehint for fairness_threshold despite it being assigned ot a float? Something like `fairness_threshold: float = 1.25`

Reply:
FINC